clients/src/main/java/org/apache/kafka/clients/admin/AlterPartitionReassignmentsOptions.java

```diff
@@ -28,4 +28,25 @@
  */
 @InterfaceStability.Evolving
 public class AlterPartitionReassignmentsOptions extends AbstractOptions<AlterPartitionReassignmentsOptions> {
+
+    private Boolean allowReplicationFactorChange = true;
+
+    /**
+     * Set the option indicating if the alter partition reassignments call should be
+     * allowed to alter the replication factor of a partition.
+     * In cases where it is not allowed, any replication factor change will result in an exception thrown by the API.
+     */
+    public boolean allowReplicationFactorChange(boolean allow) {
+        this.allowReplicationFactorChange = allow;
+        return allowReplicationFactorChange;
```

Comment:
On a quick cross-check with other *Options it appears that for such methods we return the *Options class itself. In other words, something along these lines
```
public AlterPartitionReassignmentsOptions allowReplicationFactorChange(boolean allow) {
    this.allowReplicationFactorChange = allow;
    return this;
}
```

Reply:
+1 to fluent pattern